libs/blocks/chart/chart.js

```diff
@@ -122,7 +122,8 @@ export function processMarkData(series, xUnit) {
 }
 
 export async function fetchData(link) {
-  const resp = await fetch(link.href.toLowerCase());
+  const { fetchWithCacheRules } = await import('../../utils/helpers.js');
+  const resp = await fetchWithCacheRules(link.href.toLowerCase()).catch(() => ({}));
```

Comment:
I would propose a slightly different variant
```suggestion
  const { customFetch } = await import('../../utils/helpers.js');
  const resp = await customFetch({
    resource: link.href.toLowerCase(),
    withCacheRules: true
  }).catch(() => ({}));
```